features/post.feature

```diff
@@ -8,6 +8,12 @@ Feature: Manage WordPress posts
     Then STDOUT should be a number
     And save STDOUT as {POST_ID}
 
+    When I run `wp post exists {POST_ID}`
+    Then STDOUT should be:
+      """
+    Success: Post with ID {POST_ID} exists.
+      """
+
```

Comment:
The exit code should also be tested here.

Also, you should add a test for the negative case as well.